CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v1.1.1 (2019-4-1)
+
+### Bug Fixes
+
+* TestCafe no longer crashes when the tested page's headers are larger than 8kb ([testcafe-hammerhead/#1885](https://github.com/DevExpress/testcafe-hammerhead/issues/1885))
```

Comment:
It is not true. According to problem in `testcafe-hammerhead`, we generate the `cookie` header more than 8 kb yourself. It happens only for pages with a lot cross-domain iframes.
So, the good variant is `Now TestCafe no longer crashes if the tested page contains a lot of cross-domain iframes`.